src/guides/share/nextstrain-groups.rst

```diff
@@ -126,6 +126,27 @@ Alternately, upload multiple build files at once with wildcard syntax.
 
    nextstrain remote upload s3://nextstrain-<group>/ auspice/*.json
 
+Remove files from your group
+============================
+
+You can remove specific files from your group's S3 bucket using the ``nextstrain remote delete`` command.
+For example, the following command removes your group logo and overview files.
+
+.. code-block:: bash
+
+   nextstrain remote delete \
+     s3://nextstrain-<group>/group-logo.png \
+     s3://nextstrain-<group>/group-overview.md
```

Comment:
`delete` only takes one remote path at a time. This could clearly be improved¹, but right now you'd need two invocations to delete those two objects.

¹ I think the improvement isn't allowing what's suggested here, but instead something like `nextstrain remote delete s3://nextstrain-<group> group-logo.png group-overview.md`, or once we have https://github.com/nextstrain/cli/issues/90, `nextstrain remote delete nextstrain.org/groups/<group> group-logo.png group-overview.md`.

Reply:
Thank you for reviewing this, @tsibley. I'm sorry it took so long to get back to, but 694404d reflects the correction to the remote delete command usage. Separately, 25ea448, links to the corresponding CLI commands docs.